master/buildbot/db/migrate/versions/001_initial.py

```diff
@@ -20,21 +20,22 @@
 from buildbot.db.migrate_utils import test_unicode
 from buildbot.util import json
 from buildbot.util import pickle
+from buildbot.util import sautils
 from twisted.persisted import styles
 
 metadata = sa.MetaData()
 
-last_access = sa.Table('last_access', metadata,
+last_access = sautils.Table('last_access', metadata,
                        sa.Column('who', sa.String(256), nullable=False),
```

Comment:
I think PEP8 will complain about invalid indentation of arguments here, it will expect:

```
last_access = sautils.Table('last_access', metadata,
                            sa.Column('who', sa.String(256), nullable=False),
```

Perhaps there is a way to format arguments without alignment with opening brace?

Something like:

```
last_access = sa.Table(
    'last_access', metadata,
    sa.Column('who', sa.String(256), nullable=False),
    ...
)
```

Alignment with opening brace is always bothered me with long list of arguments.